doc/source/parameters/cfd/restart.rst

```diff
@@ -0,0 +1,27 @@
+Finite element interpolation (FEM)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection controls the restart and checkpointing of the simulation. 
+This feature is used to restart a time-dependant simulation using the data written on the last checkpoint before the smulation stopped or crashed.
+The default parameters are:
+
+.. code-block:: text
+
+  subsection restart
+    set checkpoint = false
+    set restart    = false
+    set filename   = restart
+    set frequency  = 1
+  end
+
+* The ``checkpoint`` option enables checkpointing simulation informations when it is set to ``true``. You may want to enable it when launching a time-dependant simulation if you fear you might either run out of memory/simulation time during the simulation.
```

Comment:
creating a checkpoint of the simulation?

Reply:
put "You may want to enable it when launching a time-dependant simulation if you fear you might either run out of memory/simulation time during the simulation." in a tip box